libraries/eosiolib/contracts/eosio/system.hpp

```diff
@@ -1,6 +1,7 @@
 /**
  *  @file
  *  @copyright defined in eos/LICENSE
+ *  @copyright defined in cyberay/LICENSE
```

Comment:
Why 2 copyright? And typo in `cyberay/LICENSE`.

Reply:
@copyright defined in LICENSE